internal/status/status.go

```diff
@@ -199,6 +227,17 @@ func getRootStatusJson(ctx *log.Context, statusType types.StatusType, c types.Cm
 
 	return b, nil
 }
+func getRootStatusJsonWithErrorClarification(ctx *log.Context, statusType types.StatusType, c types.Cmd, msg string, indent bool, extName string, errorcode int) ([]byte, error) {
+	ctx.Log("message", "creating json to report status")
+	statusReport := types.NewStatusReportWithErrorClarification(statusType, c.Name, msg, extName, errorcode)
```

Comment:
I believe elsewhere in the code we also write status, so this would overwrite it.